ui/build

```diff
@@ -4,9 +4,12 @@ target=${1-dev}
 mode=${2-build} # build | upgrade | css | js
 echo "building ui modules with target=$target and mode=$mode"
 
-echo "node: $(node --version)"
+nodever="$(node --version)"
+echo "node: $nodever"
 echo "yarn: $(yarn --version)"
 
+[[ "$nodever" =~ ^v1[0-5]\.[0-9]+\.[0-9]+$ ]] || { echo "Node version >=10 and <16 required. Found: $nodever"; exit 1; }
```

Comment:
Wouldn't it be easier to check whether nodever is v16 rather than if it is **not** anything below 16?

Reply:
This also makes sure it's at least v10 and nothing above v16 (which ofc doesn't exist yet but who knows what might happen in the future).